packages/slate/test/models/change/without-normalization-normalize-flag-true.js

```diff
@@ -0,0 +1,49 @@
+/** @jsx h */
+
+import h from '../../helpers/h'
+
+export const normalize = true
```

Comment:
I think exporting the `normalize` here is confusing, since it doesn't appear to be used directly? Instead just the `flags` exports here would suffice I think.

Reply:
It gets used in the assert that verifies that the normalize flag got reset to the expected value

Reply:
Some tests deliberately do not specify a normalize flag in the settings to verify that the correct default of `true` is applied in the constructor and the normalize flag is `true` at the end.

Reply:
I think we need to remove it, it's too confusing. If we really need to assert that it got properly reset, we can import `assert` into this test. But I seems like something that isn't necessary? If we want that level of testing we can add unit tests later for the change methods I think.

Reply:
I'm ok with that :)